### DISCUSSION TITLE
Default Feed View Is Stupid

### DISCUSSION THREAD

**Discussion by on3al** [community/community#107587] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Literally the only thing I care about when I go to github is my active and open PRs that i've created or am a reviewer.  Get all the other shit out of the feed...  When I go to github.com the only thing I want to see at the top of my feed are 1 click links to my active PRS.. This isn't a fucking social media platform

**Comment by ericbusboom** [comment]
I really hate the new feed too; everything on it is useless to me, and it is completely wasted space, where what I really want is links to the projects that I am working on. If I spend most of the day working on three repos, then those three repos should show up in my feed, not the random stuff that is in there now.
